drf_user/serializers.py

```diff
@@ -297,3 +297,65 @@ def validate(self, attrs: dict) -> dict:
             email=attrs.get("email"), mobile=attrs.get("mobile")
         )
         return attrs
+
+
+class PasswordResetSerializer(serializers.Serializer):
+    """
+    This serializer is for password reset API.
+    otp: OTP received on your email/mobile
+    email: Email of the user whose password you're trying to reset
+    password: new password for the user
+    """
```

Comment:
Can you please change the docstring format to NumpyDoc, you can take references from the Contributing guide.